### ISSUE TITLE
Beacon monitor notification commands are not working properly

### ISSUE THREAD

**Issue by leroyloren** [home-assistant/android#3215]
**Home Assistant Android version:**
beta-3058-d1e0f29
**Android version:**
13
**Phone model:**
Pixel 7
**Home Assistant version:**
 2023.1.2
**Last working Home Assistant release (if known):**

**Description of problem:**
I am using the following scenario:

1) The phone will appear in the HOME zone
2) HA sends command_beacon_monitor - turn_on
3) automation is running in HA and waiting for beacons to be discovered

The problem is that the application apparently sets the monitoring sensor (_beacon_monitor), but in reality it does not send the "Monitoring" status to HA.

I found that just unlocking the phone (or running the app) and the sensor instantly updates in HA.

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

https://pastebin.com/zwRVd0Tw

```
{
    "message": "command_beacon_monitor",
    "data": {
        "channel": "Hlídání majáku",
        "command": "turn_on",
        "ttl": 0,
        "priority": "high"
    }
}
```
there are some attempts in the log when I send command_beacon_monitor but in HA it changes only after the phone is unlocked.

**Screenshot of problem:**

**Additional information:**

**Comment by leroyloren** [comment]
yes,

![obrazek](https://user-images.githubusercontent.com/57643470/212131767-05565878-0eb5-4746-b166-bb51546812ef.png)

**Comment by dshokouhi** [comment]
So are you not seeing Beacon updates or is this bug only about the state of the sensor changing to `Monitored`?

**Comment by leroyloren** [comment]
the beacon monitor sensor does not switch to monitoring or stopped (in ha) after sending the command. it only switches once I open the app. we have three Google Pixels in the family (5, 7, 7) and they all behave similarly.

**Comment by dshokouhi** [comment]
ok I see the issue, this was introduced after #3168 will be fixed in #3220, thanks for reporting!
